utilities/EluvioCli.js

```diff
@@ -208,7 +209,8 @@ const CmdSpaceTenantDeploy = async ({ argv }) => {
     res = await space.TenantDeploy({
       tenantName: argv.tenant_name,
       ownerAddress: argv.owner_addr,
-      adminGroupAddress: argv.tenant_admin_addr,
+      tenantAdminGroupAddress: argv.tenant_admin_addr,
```

Comment:
Same observation here would like to use plural for the groups  'tenant_admins' 'content_admins'